Sources/SwiftInspectorVisitors/TypeDescription.swift

```diff
@@ -106,9 +99,11 @@ public enum TypeDescription: Codable, Hashable {
       } else {
         return "\(parentType.asSource).\(name)<\(generics.map { $0.asSource }.joined(separator: ", "))>"
       }
+    case let .metatype(type, isType):
+      return "\(type.asSource).\(isType ? "Type" : "Protocol")"
```

Comment:
While this looks like a hack, it's inline with what [the Swift reference manual](https://docs.swift.org/swift-book/ReferenceManual/Types.html#grammar_metatype-type) says are our possible options here:
<img width="402" alt="image" src="https://user-images.githubusercontent.com/139364/140258837-ff94ca85-f16e-4633-ab01-580ebe23759d.png">